documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+- maintainable: it should be simple for maintainers to update/validate
+- without ads (readthedocs always has ads that are shown alongside our documentation)
+
+## Proposal
+
+Like for most projects the website, [fairlearn.org](http://fairlearn.org) will be the central place to look for documentation.
+From there visitors have various paths to explore content depending on what they are looking for, as detailed in the following subsections.
+
+### Homepage
+
+```
+website
+|--- Getting Started
+|--- API reference
+|--- User guide
+|--- Case studies
+|--- Community
+```
+
+### Getting Started
+
+This page provides information on
+- Installation
+- terminology (or at least a link to it)
+- where to go next, e.g. tutorials, videos, articles
+
+Installation should cover various platforms, which should be very straightforward. Any reoccurring patterns in reported issues should be listed, as well as how to troubleshoot them. It may end up similar to [this guide](https://scikit-learn.org/dev/install.html)).
+
+Example:
+
+- [pandas](https://pandas.pydata.org/getting_started.html)
+- [scikit-learn](https://scikit-learn.org/dev/getting_started.html)
+
+### API reference
+
+This is simply the generated documentation from our code. Currently we host this in readthedocs, but we want to include it on our webpage. A good example for this is [scikit-learn](https://scikit-learn.org/dev/modules/classes.html)
+
+### User guide
+
+The user guide explains all parts of Fairlearn by providing context that wouldn't fit into the code documentation such as mathematical derivations, but without using application-specific context (as we'd find it in the "case studies"). The guides are grouped by topic, e.g.
+
+1. Assessment
+    1. Metrics
+        1. ...
+    1. Visualizations
+        1. ...
+1. Mitigation
+    1. Reductions methods
+        1. Exponentiated Gradient
+        1. Grid Search
+    1. Postprocessing
+        1. Threshold Optimizer
+
+Examples:
+
+- [scikit-learn](https://scikit-learn.org/dev/user_guide.html)
+- [pandas](https://pandas.pydata.org/docs/user_guide/index.html)
+
+Of our current notebooks the following may be most suitable as "user guides":
+
+- [Group Metrics](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Group%20Metrics.ipynb) - a great example for something that should be a user guide
+- [Grid Search for Binary Classification](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Grid%20Search%20for%20Binary%20Classification.ipynb) - the purpose is mostly to show Grid Search's functionality; perhaps it may need to be trimmed down to the essentials about Grid Search
+- [Grid Search with Census Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Grid%20Search%20with%20Census%20Data.ipynb) - similar to the previous notebook it covers Grid Search; we could leverage some of this for a user guide for Grid Search, or alternatively for the dashboard visualizations
```

Comment:
We should probably just pick one of these and get rid of the other. The main difference is whether we have real data or not, and I can see arguments both ways. Basically, real data make the explanation more relatable (perhaps), but since it wouldn't be a full-blown Case Study, could also be dangerous.